roles/splunk_spark_worker/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Do we need this dependency actually? This starts both `splunk` and `spark_worker` processes in the same container, and it does not seem like we are using anything from `splunk`.

Reply:
DFS team mentioned that splunk is **_not needed_** on the containers, but it is something they requested to capture/analyze spark logs. I think it's ok to remove.